conf/nginx.conf

```diff
@@ -28,5 +28,5 @@ location {LOCATION} {
 }
 
 location @wallbag2 {
-  rewrite ^ {PATH}/app.php$is_args$args;
+  rewrite ^ {PATH}/app.php?is_args$args;
```

Comment:
I think there is a confusion about `is_args`: it's an NGINX variable, that's why there is a `$` just before - see [here](https://nginx.org/en/docs/http/ngx_http_core_module.html#var_is_args) for the official documentation.

What you've done there is to always pass `?is_args` to the `app.php` script, followed by other optional arguments. So, if no argument is given, it will call `/app.php?is_args`. But if there are ones - let's say `argument1=1&argument2=2`, it will call `/app.php?is_argsargument1=1&argument2=2`.

I'm wondering so why it is solving any 500 errors while it's *breaking* the request line... It must be something else.